CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+If you would like to contribute code to this project you can do so through GitHub by forking the repository and sending a pull request.
+
+When submitting code, please make every effort to follow existing conventions and style in order to keep the code as readable as possible.
+
+Before your code can be accepted into the project you must also sign the Individual Contributor License Agreement (CLA).
```

Comment:
Great idea with the CONTRIBUTING.md! We have no Contributor License Agreement (CLA) at the moment, is that bad? I would just remove this line.

Reply:
After further consideration, this page would not add a lot of value at the moment. We would have to put some thought into some real Contribution guidelines, otherwise it is just generic. Do I decided to close it.